doc/building.html

```diff
@@ -360,7 +360,7 @@ <h3 id="oracle-solaris-studio">Oracle Solaris Studio</h3>
 $ CC -V
 CC: Sun C++ 5.13 SunOS_i386 151846-10 2015/10/30</code></pre>
 <h3 id="microsoft-visual-studio">Microsoft Visual Studio</h3>
-<p>The minimum accepted version of Visual Studio is 2010. Older versions will not be accepted by <code>configure</code>. The maximum accepted version of Visual Studio is 2019. Versions older than 2017 are unlikely to continue working for long.</p>
```

Comment:
Please remove this hunk. This should not be affected by the change to building.md which is the master document. I will try to regenerate building.html for 11u and possibly rectify diffs in another PR.